modules/kubernetes/scripts/bootstrap.tpl

```diff
@@ -124,7 +124,7 @@ then
 else
   # You need to filter by tag Name to find the master to connect to. You don't
   # know at startup time the ip.
-  MASTER_IP=`aws ec2 describe-instances --filters "Name=tag:k8s.io/role/master,Values=1" "Name=tag:KubernetesCluster,Values=$CLUSTER_ID" --region='us-east-1' | grep '\"PrivateIpAddress\"' | cut -d ':' -f2 | cut -d'"' -f 2 | uniq`
+  MASTER_IP=`aws ec2 describe-instances --filters "Name=tag:k8s.io/role/master,Values=1" "Name=tag:KubernetesCluster,Values=$CLUSTER_ID" --region=${region} | grep '\"PrivateIpAddress\"' | cut -d ':' -f2 | cut -d'"' -f 2 | uniq`
```

Comment:
Before embedding it here I will add the mapping at the start of the file so that is clear what we are using in the script.